### PR TITLE
Removing Biobanks & BiobanksIds controllers from Directory and removing references in Analytics

### PR DESCRIPTION
**Pull request by bradthompson995**
The Biobanks & BiobanksIds api controllers are no longer neccesary so this PR removes them for the directory code and also removes the API calls from Analytics. Analytics has been modified to access the DB via local functions. Previous changes to master have done this for publications so for consistency a similar approach was used for doing the same to Analytics.

### CODE REVIEW

File: src/Analytics/AzureFunctions/Startup.cs [hunk 1]
```diff
@@ -22,7 +21,7 @@ public override void Configure(IFunctionsHostBuilder builder)
                 .GetService<IConfiguration>();
 
             // Populate connection string with credentials
-            var sqlConnection = _configuration.GetConnectionString("analyticsdb_connection");
+            var sqlConnection = _configuration.GetConnectionString("sqldb-connection");
             var sqlUsername = _configuration.GetValue("sqldb-username", "");
             var sqlPassword = _configuration.GetValue("sqldb-password", "");
```
Comment (Flemingjp) [inline src/Analytics/AzureFunctions/Startup.cs:27]: Maybe just use an `sqldb-connection` string? I know this is copied from what I wrote in the Publications function, but the Directory now parses the connection string via its deployment pipeline. Changing to that way would keep the behaviour consistent across the repo.

File: src/Analytics/Analytics/Organisation.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
+using System;
```
Comment (Flemingjp) [inline src/Analytics/Analytics/Organisation.cs:1]: Is this entity a temporary solution until we have a common `Entities` project?

**Review by Flemingjp — CHANGES_REQUESTED** [review]
Everything look good - just a question and small change suggestion that might be useful while we're here

File: src/Analytics/Analytics/Organisation.cs (hunk 2 above)
Comment (shakisparki) [reply src/Analytics/Analytics/Organisation.cs:1]: Yes it is

File: src/Analytics/AzureFunctions/Startup.cs (hunk 1 above)
Comment (Flemingjp) [reply src/Analytics/AzureFunctions/Startup.cs]: @shakisparki  Don't forget to amend the pipelines

**Review by Flemingjp — APPROVED** [review]

File: src/Analytics/Services/GoogleAnalyticsReadService.cs [hunk 3]
```diff
@@ -10,60 +10,55 @@
...
-        private readonly string DATE_FORMAT = "yyyy-MM-dd";
-
-        private readonly GoogleCredential credentials;
-        private readonly AnalyticsReportingService analytics;
+        private readonly string _viewId;
+        private readonly string _startDate; //default: "2016-01-01"; //specified in _dateFormat
+        private readonly string _dateFormat = "yyyy-MM-dd";
+        private readonly IConfiguration _config;
+
+        private AnalyticsDbContext _ctx;
```
Comment (beforan) [inline src/Analytics/Services/GoogleAnalyticsReadService.cs:34]: pretty sure this should be `readonly`; it's dependency injected and assigned in the constructor only?

**Review by beforan — APPROVED** [review]

**Review by beforan — CHANGES_REQUESTED** [review]
i misreviewed 😛 please resolve my comments 👍

**Review by beforan — APPROVED** [review]

**Review by Flemingjp — APPROVED** [review]

**Pull request merged**